themes/bootstrap3/js/facets.js

```diff
@@ -117,20 +117,217 @@ VuFind.register('facetList', function FacetList() {
 });
 
 /* --- Side Facets --- */
+var multiFacetsSelection; // Defined in results.php, here to prevent pipeline from failing
```

Comment:
Instead of defining the variable here, it might be better to add it to the global definition at the top of the file -- e.g. `/*global multiFacetsSelection, VuFind */`. That will satisfy the pipeline without requiring code changes.

Another option might be to store this setting inside the VuFind object instead of making it a global, if that's possible without creating "order of operations" challenges.